src/PostgREST/MediaType.hs

```diff
@@ -27,7 +29,28 @@ data MediaType
   | MTOctetStream
   | MTAny
   | MTOther ByteString
-  deriving (Eq)
+  | MTPlan MTPlanFormat [MTPlanOption]
+
+instance Eq MediaType where
+  MTApplicationJSON == MTApplicationJSON = True
+  MTSingularJSON == MTSingularJSON       = True
+  MTGeoJSON == MTGeoJSON                 = True
+  MTTextCSV == MTTextCSV                 = True
+  MTTextPlain == MTTextPlain             = True
+  MTTextXML == MTTextXML                 = True
+  MTOpenAPI == MTOpenAPI                 = True
+  MTUrlEncoded == MTUrlEncoded           = True
+  MTOctetStream == MTOctetStream         = True
+  MTAny == MTAny                         = True
+  MTOther bs1 == MTOther bs2             = bs1 == bs2
+  MTPlan {} == MTPlan {}                 = True
+  _ == _                                 = False
```

Comment:
Added this for keeping the `mutuallyAgreeable` function working for the MTPlan. It's kinda icky but I didn't saw a better way without changing too much code.

Reply:
@laurenceisla Pointed me out [to this technique](https://stackoverflow.com/a/5956095/4692662) so hacking the MediaType Eq is no longer necessary.